File: Lecture4/Ex43_exercises.java
public class Ex43_exercises {
    public static void main(String[] args) {

        double a, b, c, d;

        a = Math.atan(Math.sin(Math.toRadians(32)) * Math.cos(Math.toRadians(32)));

        b = Math.pow(Math.pow(Math.E, 1.67) * Math.pow(1.65, Math.log(8)/Math.log(20)), 1.0/4.0);

        c = Math.log(Math.pow(21, Math.log(35)))/Math.log(7);

        d = Math.exp(Math.sqrt(Math.log(4))+Math.sqrt(Math.log(5)));

        System.out.println(a +" "+ b +" "+ c +" "+ d);
    }
}

File: Lecture4/Ex43_exercises.c
#include <stdio.h>
#include <math.h>

double toRadians(double degree) {
    return degree / 180.0 * M_PI;
}

int main(void) {
    double a, b, c, d;

    a = atan(sin(toRadians(32)) * cos(toRadians(32)));

    b = pow(pow(M_E, 1.67) * pow(1.65, log(8)/log(20)), 1.0/4.0);

    c = log(pow(21, log(35)))/log(7);

    d = exp(sqrt(log(4))+sqrt(log(5)));

    printf("%lf %lf %lf %lf", a, b, c, d);

    return 0;
}
